.github/workflows/connector-node-integration.yml

```diff
@@ -0,0 +1,130 @@
+name: Connector Node Integration tests
+
+on:
```

Comment:
Perhaps a condition should be added here, which will only be triggered when the connector-node changes.